content/index.rst

```diff
@@ -4,173 +4,126 @@ grid_template: true
     :url: '/contact/'
 
-    Get a free consultation
+    Schedule a consultation
 
-  .. _web application design and development: /work/
+  .. _`web application strategy, design, and development`: /work/
 
 
-Software for Humans
-===================
+OddBird
+=======
 
 
 .. callmacro:: content.macros.j2#get_quotes
-  :slug: 'work/coachhub'
-  :index: 1
-
+  :page: 'work/coachhub'
+  :slug: 'remote'
 
 
 .. -------------------------------------
 .. callmacro:: content.macros.j2#divider
-.. callmacro:: projects/splash.macros.j2#splash_list
-
-.. ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-.. callmacro:: content.macros.j2#rst
-  :tag: 'start'
-
-Small Team, Large Expertise
----------------------------
-
-Partnering with funded startups and enterprise companies
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-OddBird is a small team,
-made up of industry leaders
-across a range of design and code expertise,
-and working together for years
-with a tight-knit, agile, and collaborative process.
-We're able to give every project complete focus,
-putting the needs of your company and users first,
-while integrating design and development from start to finish.
-Whether you're looking for staff-augmentation,
-or a custom application,
-we provide *big agency skill,
-with the attention, communication, and agility
-of an internal team.*
-
-.. callmacro:: content.macros.j2#link_button
-  :url: '/contact/'
-
-  Start a conversation
-
-.. callmacro:: content.macros.j2#rst
-  :tag: 'end'
-.. ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. -------------------------------------
 
-.. callmacro:: content.macros.j2#get_quotes
-  :slug: 'work/quarqnet'
-  :index: 1
 
+.. callmacro:: projects/splash.macros.j2#splash_list
+  :headline: 'Featured Partners'
 
 
 .. -------------------------------------
 .. callmacro:: content.macros.j2#divider
-.. callmacro:: projects/splash.macros.j2#splash_list
-  :slugs: [
-    'susy/index',
-    'open-source/django',
-    'true/index',
-    'open-source/plone'
-    ]
-
-.. ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-.. callmacro:: content.macros.j2#rst
-  :tag: 'start'
-
-Open Source Leadership
-----------------------
-
-Experts in Code Architecture & Tooling
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-Founded by a `Django core developer`_
-and the `creator of Susy`_ responsive-layout framework,
-open-source software has always been central
-to the OddBird vision.
-Whether we're building tools for developers like ourselves,
-or third-party consumers,
-we keep the needs of real people
-at the center of everything we do.
+.. -------------------------------------
 
-.. callmacro:: content.macros.j2#link_button
-  :url: '/contact/'
 
-  Hire us for a workshop or conference talk
+.. callmacro:: content.macros.j2#get_quotes
+  :page: 'work/medcurbside'
+  :slug: 'goals'
+
+.. callmacro:: content.macros.j2#image_block
+  :image: '/static/images/projects/trig-mobile.jpg'
+  :headline: 'Are We The Right Fit For You?'
+
```

Comment:
Looks like these isn't a blue line above "Are We the Right Fit for You?" and I think that looks nice. I'd recommend taking about the blue line below as well as the blue lines above and below the "Industry-Leading Expertise" section too.

Reply:
I see a blue line above and below both quotes.

![screenshot 2017-10-30 15 47 43](https://user-images.githubusercontent.com/552316/32192154-b59756ea-bd89-11e7-84e4-a194a08c7e28.jpg)

Reply:
There is a gradient-linen in all 4 locations - not sure why one isn't showing up for you. I thought they added focus, but we can remove them.